packages/maskbook/src/web3/UI/EthereumStatusBar.tsx

```diff
@@ -1,13 +1,20 @@
 import { Box, makeStyles, createStyles, Theme, ChipProps, BoxProps } from '@material-ui/core'
 import { useStylesExtends } from '../../components/custom-ui-helper'
 import { EthereumAccountButton } from './EthereumAccountButton'
+import { EthereumGasButton } from './EthereumGasButton'
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
         root: {
             display: 'flex',
             alignItems: 'center',
             justifyContent: 'flex-end',
+            '& > * ': {
+                marginRight: theme.spacing(1),
+            },
+            '& > * : last-child': {
```

Comment:
I think too much global selector is an anti-pattern. Please pay attention to it.

Reply:
Acutally, `& > * : last-child` it an invalid selector.